src/main.cpp

```diff
@@ -373,6 +381,17 @@ int main(int argc, char **argv)
 			{
 				g_nNewNice = nNewNice;
 			}
+			if ( g_bRt )
+			{
+				int sched_prio = sched_get_priority_min(SCHED_RR);
+				struct sched_param sched = {.sched_priority = sched_prio};
+				if (pthread_getschedparam(pthread_self(), &g_nOldPolicy, &g_schedOldParam)) {
```

Comment:
Nit: 0 can be used instead of `pthread_self` here

Reply:
Are you sure? I get a segfault when I do this and `pthread_getschedparam(3)` does not say anything about this.

Reply:
Hm, not sure where I found that, I can't find it anymore. Let's stick to `pthread_self` then.

Reply:
Ah, I mixed it up with `sched_getparam`. `pthread_getschedparam` is different and doesn't accept 0.